devtools/steam_achivements.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# This script takes two parameters - the first is the steam game id, the second (optional) is the scummvm
+# game id. E.g.
+#
+# ./steam_achivements.py 212050 resonance
+#
+# It will extract the steam stats information to create the scummvm achievement tables with the macros found
+# in AGS achievements_table.h
+
+import requests
+from requests_html import HTMLSession
+import sys
+
+steam_game_id = sys.argv[1:]
+scummvm_game_id = sys.argv[2:]
```

Comment:
these probably should be below the length check below?

Reply:
would look saner, yes - but it works like this due to the `:`

Reply:
Fixed